Scripts/update-translations.rb

```diff
@@ -124,27 +121,52 @@
   langs = ALL_LANGS
 end
 
+script_root = __dir__
+project_dir = File.dirname(script_root)
+
 langs.each do |code,local|
-  lang_dir = File.join('WordPress', 'Resources', "#{local}.lproj")
-  puts "Updating #{code}"
-  system "mkdir -p #{lang_dir}"
+  lang_dir = File.join(project_dir, 'WordPress', 'Resources', "#{local}.lproj")
+  puts "Updating #{code} in #{lang_dir}"
+  system "mkdir", "-p", lang_dir
 
-  # Download for production
-  system "if [ -e #{lang_dir}/Localizable#{strings_file_ext}.strings ]; then cp #{lang_dir}/Localizable#{strings_file_ext}.strings #{lang_dir}/Localizable#{strings_file_ext}.strings.bak; fi"
+  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
+  backup_destination = "#{destination}.bak"
+
+  # Step 2 – Download the new strings
+  if File.exist? destination
+    FileUtils.copy destination, backup_destination
+  end
 
   url = "#{download_url}/#{code}/default/export-translations?#{strings_filter}format=strings"
-  destination = "#{lang_dir}/Localizable#{strings_file_ext}.strings"
 
-  system "curl -fLso #{destination} \"#{url}\"" or begin
+  system "curl", "-fgsLo", destination, url or begin
     puts "Error downloading #{code}"
   end
```

Comment:
For readability I'd add parentheses around arguments in that case, and/or just avoid the `begin/end` block given it's a one-liner
```suggestion
  system("curl", "-fgsLo", destination, url) || puts "Error downloading #{code}"
```

Though not sure what's `#{code}` is supposed to be, I think we won't capture the error code as expected (I think we'd need to resort to using `Open3.popen3` to capture the error). So overall and given we know we'll refactor that sonn-ish anyway, maybe just use the url in the error message instead.

```suggestion
  system("curl", "-fgsLo", destination, url) || puts "Error downloading #{url}"
```